### PR TITLE
Messages

### LINKED ISSUE #133: [Feature] Move More Messages
# Feature request: Code

>> Move more messages to the Messages class

## What would this break

- Nothing

### PR DESCRIPTION
**Pull request by Lukamon24**
#133

### CODE REVIEW

File: src/main/java/org/zone/utils/Messages.java [hunk 1]
```diff
@@ -329,4 +330,28 @@ public static Component getEnabledTag() {
     public static Component getEnabledInfo(boolean check) {
         return getEnabledTag().append(Component.text(check).color(NamedTextColor.GOLD));
     }
+
+    @SuppressWarnings("TypeMayBeWeakened")
```
Comment (mosemister) [inline src/main/java/org/zone/utils/Messages.java]: Dont suppress unless you need to

Comment (Lukamon24) [reply src/main/java/org/zone/utils/Messages.java]: uhm well it just gives weak warnings..

File: src/main/java/org/zone/commands/structure/ZoneCommands.java [hunk 2]
```diff
@@ -77,6 +78,7 @@
     ZoneFlagGreetingsViewCommand ZONE_FLAG_GREETINGS_VIEW_COMMAND = new ZoneFlagGreetingsViewCommand();
     ZoneFlagInteractItemframesViewCommand ZONE_FLAG_INTERACT_ITEMFRAMES_VIEW_COMMAND = new ZoneFlagInteractItemframesViewCommand();
 
+    EditBoundsStartCommand EDIT_BOUNDS_START_COMMAND = new EditBoundsStartCommand();
```
Comment (mosemister) [inline src/main/java/org/zone/commands/structure/ZoneCommands.java]: I wouldnt register the command unless you have a end command to go with it. Dont want to add a half done job

Comment (Lukamon24) [reply src/main/java/org/zone/commands/structure/ZoneCommands.java]: ok XD

**Review by mosemister — APPROVED** [review]
Small PR, but ok

**Review by mosemister — CHANGES_REQUESTED** [review]
Wrong branch, should be master 2 currently

**Comment by Lukamon24** [comment]
eh?

**Review by mosemister — APPROVED** [review]
Small change, but ok

**Pull request merged**